test/registry_test.rb

```diff
@@ -24,32 +24,19 @@ class ChildTestRegistry < TestRegistry
     attr_reader :registry, :child_registry
 
     test ".factory defines fixture factories" do
-      block = proc { factory(:recipe_fixtury, class_name: "Recipe") }
+      block = proc { factory(:recipe_fixtury, class: -> { Recipe }) }
       registry.define_factories(&block)
       assert_nothing_raised do
         assert_instance_of FixtureFactory::Definition, registry.all_factory_definitions.fetch(:recipe_fixtury)
       end
     end
 
-    test ".factory works with deprecated class: option" do
```

Comment:
We need a test here like this for `class_name` since this is where we still support it.